### PR TITLE
Fix ordering of before/after in print-pretty-timed-*

### LINKED ISSUE #8158: Coq makefile: print-pretty-timed-diff mixes up "before" and "after"
#### Version
8.8

#### Operating system
NixOS

#### Description of the problem
The makefile template passes arguments into `make-both-time-files.py` in the opposite order they should be in. [From `CoqMakefile.in`](https://github.com/coq/coq/blob/a44b4be1bf8b96e941216cd10cfb5981a825c3fa/tools/CoqMakefile.in#L346):
```make
COQMAKE_BOTH_TIME_FILES ?= "$(COQLIB)/tools/make-both-time-files.py"
print-pretty-timed-diff::
$(HIDE)$(COQMAKE_BOTH_TIME_FILES) --sort-by=$(TIMING_SORT_BY) $(TIME_OF_BUILD_BEFORE_FILE) $(TIME_OF_BUILD_AFTER_FILE) $(TIME_OF_PRETTY_BOTH_BUILD_FILE) $(TIME_OF_PRETTY_BUILD_EXTRA_FILES)
```
but [from `make-both-time-files.py`](https://github.com/coq/coq/blob/a44b4be1bf8b96e941216cd10cfb5981a825c3fa/tools/make-both-time-files.py#L6):
```python
 USAGE = 'Usage: %s [--sort-by=auto|absolute|diff] AFTER_FILE_NAME BEFORE_FILE_NAME [OUTPUT_FILE_NAME ..]' % sys.argv[0]
```
cc: @JasonGross

### PR DESCRIPTION
**Pull request by JasonGross**
**Kind:** bug fix / infrastructure.

Fixes / closes #8158

- [x] Added / updated test-suite
- [x] Entry added in CHANGES.

I can't seem to get the makefile test-suite to build at all locally (is it broken on master?), so we should make sure the test-suite passes on the CI before merging this.

### CODE REVIEW

File: tools/CoqMakefile.in [hunk 1]
```diff
@@ -343,19 +343,19 @@ make-pretty-timed make-pretty-timed-before make-pretty-timed-after::
 print-pretty-timed::
 	$(HIDE)$(COQMAKE_ONE_TIME_FILE) $(TIME_OF_BUILD_FILE) $(TIME_OF_PRETTY_BUILD_FILE) $(TIME_OF_PRETTY_BUILD_EXTRA_FILES)
 print-pretty-timed-diff::
-	$(HIDE)$(COQMAKE_BOTH_TIME_FILES) --sort-by=$(TIMING_SORT_BY) $(TIME_OF_BUILD_BEFORE_FILE) $(TIME_OF_BUILD_AFTER_FILE) $(TIME_OF_PRETTY_BOTH_BUILD_FILE) $(TIME_OF_PRETTY_BUILD_EXTRA_FILES)
+	$(HIDE)$(COQMAKE_BOTH_TIME_FILES) --sort-by=$(TIMING_SORT_BY) $(TIME_OF_BUILD_AFTER_FILE) $(TIME_OF_BUILD_BEFORE_FILE) $(TIME_OF_PRETTY_BOTH_BUILD_FILE) $(TIME_OF_PRETTY_BUILD_EXTRA_FILES)
 ifeq (,$(BEFORE))
 print-pretty-single-time-diff::
-	@echo 'Error: Usage: $(MAKE) print-pretty-single-time-diff BEFORE=path/to/file.v.before-timing AFTER=path/to/file.v.after-timing'
+	@echo 'Error: Usage: $(MAKE) print-pretty-single-time-diff AFTER=path/to/file.v.after-timing BEFORE=path/to/file.v.before-timing'
```
Comment (gares) [inline tools/CoqMakefile.in]: Is this change relevant?

**Comment by gares** [comment]
I'm a bit lost. You pass `NAME=value` as arguments to the script and the order matters? So what is the name for?

File: tools/CoqMakefile.in (hunk 1 above)
Comment (JasonGross) [reply tools/CoqMakefile.in]: No, this change is not relevant, and nor is the other `echo` line.  But I figured someone reading the source code would parse it more easily if I always pass the "after" file first.  (Note that the other two changes in this file pass things by position, not by name.)

**Comment by JasonGross** [comment]
Two of the changes are cosmetic (the ones where we tell the user to pass `NAME=value` to `make`), and the other two are meaningful (the ones where we pass positional values to the scripts).  Does this clear things up?

**Comment by gares** [comment]
test suite fails

**Comment by JasonGross** [comment]
Test-suite should now succeed (or, if it fails, as it does on my machine, on `coq-makefile/findlib-package.log` with `File not found on loadpath : ltac_plugin.cm[ao]`, it should fail alike on this PR and on master).

**Comment by Zimmi48** [comment]
That's probably because you don't run configure with `-profile devel` but another set of option that doesn't contain `-local`.

**Comment by Zimmi48** [comment]
Ah sorry I hadn't seen that you do the `make install` step in #8318.

**Review by gares — APPROVED** [review]

**Comment by Zimmi48** [comment]
@JasonGross I'm backportng this PR to v8.8 but this required me to move the CHANGES entry to the appropriate section (see https://gitlab.com/coq/coq/commit/5f090e579183dade02fe2d85a3c371d39ae3b7f7). Please open a PR doing the same in master.

**Pull request merged**